src/serialize.jl

```diff
@@ -0,0 +1,18 @@
+const pickle = PyNULL()
+
+function get_pickle()
+    if pickle.o == C_NULL
+        copy!(pickle, pyimport("pickle"))
+    end
+
+    return pickle
+end
+
+function Base.serialize(s::AbstractSerializer, pyo::PyObject)
+    Base.serialize_type(s, PyObject)
+    serialize(s, get_pickle()[:dumps](pyo))
```

Comment:
In low-level functions like this you want to avoid the automatic type detection.  You could do `pycall(get_pickle()["dumps"], String, pyo)`.

However, it would be even better to use the low-level buffer interface here.  That way you can avoid copying the data entirely, and work with the raw byte array.